etc/eod-config/eod-config.yml

```diff
@@ -59,9 +59,21 @@ eod_euclid_project:
   description: EUCLID Consortium
   project_domain: default
   user_domain: default
-  users: "{{ eod_resops_users }}"
+  users: "{{ eod_resops_users }} + {{ eod_euclid_users }}"
```

Comment:
Not seen it like this before, I normally use `{{ eod_resops_users + eod_euclid_users }}`. No problem if it works though.

Reply:
seemed to but the single set of curly braces looks nicer.